components/admin_console/admin_definition.jsx

```diff
@@ -5626,18 +5584,31 @@ const AdminDefinition = {
+                        help_text_default: 'Enables channel sidebar organization options in **Account Settings > Sidebar > Channel grouping and sorting** including options for grouping unread channels, sorting channels by most recent post and combining all channel types into a single list.',
                         help_text_markdown: true,
+                        isHidden: it.configIsFalse('ServiceSettings', 'EnableLegacySidebar'),
                         isDisabled: it.not(it.userHasWritePermissionOnResource('experimental')),
                     },
                     {
                         type: Constants.SettingsTypes.TYPE_BOOL,
-                        key: 'DisplaySettings.ExperimentalTimezone',
-                        label: t('admin.experimental.experimentalTimezone.title'),
-                        label_default: 'Timezone:',
-                        help_text: t('admin.experimental.experimentalTimezone.desc'),
-                        help_text_default: 'Select the timezone used for timestamps in the user interface and email notifications. When true, the Timezone setting is visible in the Account Settings and a time zone is automatically assigned in the next active session. When false, the Timezone setting is hidden in the Account Settings.',
+                        key: 'TeamSettings.EnableXToLeaveChannelsFromLHS',
+                        label: t('admin.experimental.enableXToLeaveChannelsFromLHS.title'),
+                        label_default: 'Enable X to Leave Channels from Left-Hand Sidebar:',
+                        help_text: t('admin.experimental.enableXToLeaveChannelsFromLHS.desc'),
+                        help_text_default: 'When true, users can leave Public and Private Channels by clicking the “x” beside the channel name. When false, users must use the **Leave Channel** option from the channel menu to leave channels.',
+                        help_text_markdown: true,
+                        isHidden: it.configIsFalse('ServiceSettings', 'EnableLegacySidebar'),
+                        isDisabled: it.not(it.userHasWritePermissionOnResource('experimental')),
+                    },
+                    {
+                        type: Constants.SettingsTypes.TYPE_BOOL,
+                        key: 'ServiceSettings.CloseUnusedDirectMessages',
+                        label: t('admin.experimental.closeUnusedDirectMessages.title'),
+                        label_default: 'Autoclose Direct Messages in Sidebar:',
+                        help_text: t('admin.experimental.closeUnusedDirectMessages.desc'),
+                        help_text_default: 'When true, direct message conversations with no activity for 7 days will be hidden from the sidebar. When false, conversations remain in the sidebar until they are manually closed.',
```

Comment:
this should probably be removed when we implement the limit visible dms by the user because we will get rid of the autoclose logic. Just adding a comment to keep in mind for myself :)

Reply:
We still need it for the old sidebar, so it can't be removed until that goes.